src/config.ts

```diff
@@ -0,0 +1,14 @@
+import { Type } from "@sinclair/typebox";
+import { Value } from "@sinclair/typebox/value";
+
+const envSchema = Type.Object({
+  POSTGRES_DB: Type.String(),
+  POSTGRES_USER: Type.String(),
+  POSTGRES_PASSWORD: Type.String(),
+  POSTGRES_HOST: Type.String(),
+  POSTGRES_PORT: Type.String(),
+  JWT_SECRET: Type.String(),
+})
+// TODO: this is ugly, find a better way to do this
```

Comment:
It feels bad that it's in two steps. I've used the t3-env one that's built with zod and it felt much better. I think it's super similar to env-schema, we can move to that in a separate PR for sure.

Reply:
They're almost identical. I would love to try zod but from my research TypeBox is superior

Reply:
Typebox is definitely faster and has more features, so that's what I'd use in a performance oriented production service. So env-schema is a nobrainer here

Zod is easier to use (at least to me) so that's my choice of small pet projects.